packages/transformer/src/IModelTransformer.ts

```diff
@@ -1264,12 +1264,10 @@ export class IModelTransformer extends IModelExportHandler {
     );
   }
 
-  /** Returns `true` if *brute force* delete detections should be run.
+  /** Returns `true` if *brute force* delete detections should be run. This is only called if [[IModelTransformerOptions.forceExternalSourceAspectProvenance]] option is true
```

Comment:
```suggestion
  /** Returns `true` if *brute force* delete detections should be run.
   * @note This is only called if [[IModelTransformerOptions.forceExternalSourceAspectProvenance]] option is true
```

Reply:
FYI, I realized its IModelTransformOptions not IModelTransformerOptions. Changed it otherwise though, resolving and merging.